packages/@ama-sdk/create/src/index.it.spec.ts

```diff
@@ -8,8 +8,8 @@ const o3rEnvironment = globalThis.o3rEnvironment;
 
 import {
   getDefaultExecSyncOptions,
-  getPackageManager,
   getYarnVersionFromRoot,
+  isYarnTest,
```

Comment:
are you sure this is correct ?
I believe you should use `o3rEnvironment.testEnvironment.isYarnTest` inside the test

Reply:
you are correct